client/src/components/organisms/DonorImpactSection.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FunctionComponent, useState } from "react";
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+
+import MapWithMarkers, { MapWithMarkersPoint } from '../atoms/MapWithMarkers'
+import { Statistic, Testimonial } from '../../data/types/donorHomepageConfig'
+import BaseMap from '../../assets/kw-region-map.png'
+import Card from '../atoms/CardWithShadow'
+import Cursor from '../../assets/cursor.png'
+import DonorHomepageConfig from '../../config/donorHompageConfig.json'
+
+const DonorImpactSection: FunctionComponent = () => {
+  const [selectedTestimonial, _setSelectedTestimonial] = useState<Testimonial | null>(null);
```

Comment:
Nit, _ convention isn't in-line with what we've been doing so far

Reply:
Maybe call the setSelectedTestimonial function selectTestimonial or something like that

Reply:
_ might not be what we're doing, but i only recently found in other code.
imo we should use it going forward because it's obvious that `_setSelectedTestimonial` on it's own